packages/core/package.json

```diff
@@ -49,6 +49,7 @@
 		"@types/react": "*",
 		"resolve-tspaths": "*",
 		"tsc-watch": "*",
-		"typescript": "*"
+		"typescript": "*",
+		"html-react-parser": "^5.1.12"
```

Comment:
Not entirely sure how this works, do we want to manage this separately from next, or could we use a "*" since we only care about the type? 

cc @ayushnirwal